src/Persistance/RepositoryImplementations/SystemRegisterRepository.cs

```diff
@@ -58,17 +59,19 @@ public async Task<List<RegisteredSystem>> GetAllActiveSystems()
     }
 
     /// <inheritdoc/>  
-    public async Task<Guid?> CreateRegisteredSystem(RegisteredSystem toBeInserted)
+    public async Task<Guid?> CreateRegisteredSystem(RegisteredSystem toBeInserted, string[] defaultRights)
     {
         const string QUERY = /*strpsql*/@"
             INSERT INTO altinn_authentication_integration.system_register(
                 registered_system_id,
                 system_vendor,
-                friendly_product_name)
+                friendly_product_name,
```

Comment:
is the field name same as this in table?

Reply:
yes it is